src/argus_htmx/themes/utils.py

```diff
@@ -1,17 +1,42 @@
+from importlib.resources import files
+
 from django.conf import settings
-from argus_htmx import settings as argus_htmx_settings
+from django.core.exceptions import ImproperlyConfigured
 
+from argus_htmx import settings as default_htmx_settings
 
-def get_themes():
-    return getattr(settings, "DAISYUI_THEMES", argus_htmx_settings.DAISYUI_THEMES)
 
+__all__ = ["get_theme_names"]
 
-def get_theme_names():
-    themes = get_themes()
+
+def get_themes_from_setting():
+    themes_setting = getattr(settings, "DAISYUI_THEMES", default_htmx_settings.DAISYUI_THEMES)
     theme_names = []
-    for theme in themes:
+    for theme in themes_setting:
         if isinstance(theme, str):
             theme_names.append(theme)
         elif isinstance(theme, dict):
             theme_names.extend(theme.keys())
     return theme_names
+
+
+def get_themes_from_css():
+    styles_css = files("argus_htmx").joinpath("static/styles.css").read_text()
+    theme_names = []
+    for line in styles_css:
+        if "data-theme_name=" not in line:
+            continue
```

Comment:
css file may be minified and have everything on one line

Reply:
This file is the one delivered with the code. Not much to gain in minifing it. The ones used by the servers (after collectstatic) is another matter but our python code should not assume that code is accessible in any way.

Reply:
I'm not suggesting to minify the current default argus styles.css, I'm saying that we have a minified styles.css running in deployment, at non-default location (see also my comment about path_to_stylesheet). We would get errors if we try to start argus in deployment.